tools/pot/openvino/tools/pot/graph/passes.py

```diff
@@ -489,6 +490,52 @@ def disconnect_fq_node(self, fq_node):
         if parent_node.type == 'Const':
             parent_node['need_shape_inference'] = True
 
+    def optimize_for_gp_hw(self, graph, target_device):
+        """
+        Removing redundant FQs before operation Add
+        for SPR(CPU) and ATS(GPU) platform
+        """
+        def _walk_for_branch(node):
+            input_node = node
+            delete_const = lambda node: ([op for op in node if op is not None and op.type != 'Const'])
+            while True:
+                input_node = get_node_inputs(input_node)
+                input_node = delete_const(input_node)
+                if len(input_node) > 1:
+                    return False
+                input_node = input_node[0]
+                if input_node.type in ['Convolution', 'GroupConvolution', 'MatMul']:
```

Comment:
Shouldn't this condition be extended with ConvolutionBackpropData (aka Deconvolution)?

Reply:
@KodiaqQ in [documentation LPT](https://github.com/openvinotoolkit/openvino/blob/master/docs/IE_PLUGIN_DG/plugin_transformation_pipeline/low_precision_transformations/transformations/step3_main/arithmetic/add.md) not talking about 'ConvolutionBackpropData'